From 47eabc562a8883d757e3cff487c6c6e98def5a21 Mon Sep 17 00:00:00 2001
From: Andrew Svetlov <andrew.svetlov@gmail.com>
Date: Fri, 28 May 2021 12:42:17 +0300
Subject: [PATCH] Drop tests for semicolon as a separator

yarl 1.6.3 ptests check the use of semicolons as separators in URLs,
which are no longer valid by default in newer versions of Python. This
patch is in python3-yarl master, but there is no release with it yet, so
backport it until there is a new release.

For more info, see: 
https://bugs.python.org/issue42967
https://github.com/aio-libs/yarl/issues/563

Upstream-Status: Backport
(https://github.com/aio-libs/yarl/commit/47eabc562a8883d757e3cff487c6c6e98def5a21)

Signed-off-by: Trevor Gamblin <trevor.gamblin@windriver.com>

---
 tests/test_url_query.py | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/tests/test_url_query.py b/tests/test_url_query.py
index d2ca640..07f858b 100644
--- a/tests/test_url_query.py
+++ b/tests/test_url_query.py
@@ -60,14 +60,3 @@ def test_ampersand_as_value():
     u = URL("http://127.0.0.1/?a=1%26b=2")
     assert len(u.query) == 1
     assert u.query["a"] == "1&b=2"
-
-
-def test_semicolon_as_separator():
-    u = URL("http://127.0.0.1/?a=1;b=2")
-    assert len(u.query) == 2
-
-
-def test_semicolon_as_value():
-    u = URL("http://127.0.0.1/?a=1%3Bb=2")
-    assert len(u.query) == 1
-    assert u.query["a"] == "1;b=2"
-- 
2.31.1

